cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -126,7 +140,7 @@ private StandaloneMain(List<Module> modules, CConfiguration cConf) {
     injector = Guice.createInjector(modules);
 
     // Start ZK client and Kafka client only when audit is enabled
-    boolean auditEnabled = cConf.getBoolean(Constants.Audit.ENABLED);
+    auditEnabled = cConf.getBoolean(Constants.Audit.ENABLED);
```

Comment:
Why need to convert it to field? We already have the `zkClient` and `kafkaClient` instances. We can just check for null in the start/stop methods.